docs/google/calibration.md

```diff
@@ -0,0 +1,170 @@
+# Calibration Metrics
+
+Processors that are quantum computers periodically undergo calibrations to
+maintain the quality of the programs that can be run on these processors.
+During this calibration metrics about the performance of the quantum computer 
+are collected.  This calibration data is stored by Quantum Engine and users can
+then query for the current or previous state of the calibration.
+Calibrations are also available for past jobs.
+
+Calibration objects are a dictionary from metric name (see below) to the value
+of the metric.  Note that the value of the metric is also usually a dictionary
+(for instance, from qubit or qubit pair to a float value).
+
+## Retrieving calibration metrics
+
+Calibration metrics can be retrieved using an engine instance or with a job.
+
+```
+import cirq.google as cg
+
+# Create an Engine object to use.
+# Replace YOUR_PROJECT_ID with the id from your cloud project.
+engine = cg.Engine(project_id=YOUR_PROJECT_ID, proto_version=cg.ProtoVersion.V2)
+
+# TODO(#2720): this PR refactors some of these functions...
+
+# gets the latest calibration metrics
+latest_calibration = engine.get_latest_calibration()
+
+# If you know the timestamp of a previous calibration, you can retrieve the
+# calibration using the timestamp
+calibration_name = ("projects/" + PROJECT_ID 
+                    + "/processors/" + PROCESSOR_ID
+                    + "/calibrations/" 
+                    + str(EPOCH_TIMESTAMP_SECONDS)) 
```

Comment:
It's a bit unfortunate that we expose REST protocol to the users like that.

Reply:
I think PR 2720 should help alleviate some of this.